### ISSUE TITLE
Show label for hashtag subscriptions in home timeline

### ISSUE THREAD

**Issue by timdmackey** [mastodon/mastodon#22540]
### Pitch

When someone boosts a post into your timeline, it's helpful context that a label is applied letting you know who boosted it, so you know why the post is in your timeline. The ability for apps to show a similar label for hashtag subscriptions would be equally useful.

This is what a boosted post looks like in the Mastodon web UI:
<img width="788" alt="Screen Shot 2022-12-20 at 1 09 58 AM" src="https://user-images.githubusercontent.com/182772/208628964-479a41cf-36f7-4746-be6f-dd90428f9db0.png">
And this is how a hashtag subscription post might look:
<img width="790" alt="Screen Shot 2022-12-20 at 1 06 19 AM" src="https://user-images.githubusercontent.com/182772/208628982-ac904bd2-34e9-4e24-9ab3-d6dae93aaf4d.png">

### Motivation

I often find myself reading a post in my timeline and wondering "who is this person, why am I seeing their posts? Do I have the federated timeline open?" I then realize that the person has randomly decided to include a hashtag in their post that I'm subscribed to, and the hashtag isn't relevant to the post (or I just didn't realize it was relevant).

Having the ability to label posts that originate from a hashtag subscription would be a helpful context clue for managing one's home timeline. I'm hoping that this can be added to the data available to all apps so that this kind of labelling is an option.

**Comment by DagAgren** [comment]
This originally came as a feature request for Toot!, but I don't think the current API gives enough data to do it. However, it seems like a very obvious user experience improvement.

**Comment by rbairwell** [comment]
I think this is a duplicate of [#22444](https://github.com/mastodon/mastodon/issues/22444) which itself is a duplicate of [#19895](https://github.com/mastodon/mastodon/issues/19895)

**Comment by timdmackey** [comment]
> I think this is a duplicate of [#22444](https://github.com/mastodon/mastodon/issues/22444) which itself is a duplicate of [#19895](https://github.com/mastodon/mastodon/issues/19895)

You're right, I didn't see those before posting this. Since my original intention was for this to be available in the API, I'm going to rewrite the post to reflect that.
